recipes-initramfs/iot2050-initramfs-ftpm-hook/iot2050-initramfs-ftpm-hook_0.1.bb

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) Siemens AG, 2023
+#
+# Authors:
+#  Su Bao Cheng <baocheng.su@siemens.com>
+#
+# SPDX-License-Identifier: MIT
+#
+
+inherit dpkg-raw
+
+SRC_URI += " \
+    file://ftpm.hook \
+    file://ftpm.script \
+    "
+
+DEBIAN_DEPENDS = "initramfs-tools, tee-supplicant"
+
+do_install[cleandirs] += " \
+    ${D}/usr/share/initramfs-tools/hooks \
+    ${D}/usr/share/initramfs-tools/scripts/local-bottom"
+
+do_install() {
+    install -m 0755 "${WORKDIR}/ftpm.hook" \
+        "${D}/usr/share/initramfs-tools/hooks/ftpm"
+    install -m 0755 "${WORKDIR}/ftpm.script" \
+        "${D}/usr/share/initramfs-tools/scripts/local-bottom/ftpm"
+}
```

Comment:
Subject should be: "Add recipe **for** optee based ftpm hook".

And this is isar-cip-core material, nothing IOT2050-specific.

Reply:
Acked for the subject refining.

For make it into isar-cip-core, also need to upstream the tee-supplicant. WIP.

Reply:
https://lore.kernel.org/cip-dev/a269dde9f2e7bf0412eccc8f453e264549cb5d0e.camel@siemens.com/T/#u
https://lore.kernel.org/cip-dev/664ef5ea0fa832ff709718b249a93864f98714c6.camel@siemens.com/T/#u

-- My git send email is broken so I sent them via evolution...